controllers/volumereplicationgroup_controller.go

```diff
@@ -1146,6 +1151,62 @@ func (v *VRGInstance) updateVRGLastGroupSyncTime() {
 	v.instance.Status.LastGroupSyncTime = leastLastSyncTime
 }
 
+func (v *VRGInstance) updateVRGLastGroupSyncDuration() {
+	var maxLastSyncDuration *metav1.Time
+
+	for _, protectedPVC := range v.instance.Status.ProtectedPVCs {
+
+		// If any protected PVC reports nil, report that back (no sync duration available)
+		if protectedPVC.LastSyncDuration == nil {
+			maxLastSyncDuration = nil
+
+			break
+		}
+
+		if maxLastSyncDuration == nil {
+			maxLastSyncDuration = protectedPVC.LastSyncDuration
+
+			continue
+		}
+
+		if protectedPVC.LastSyncDuration != nil && protectedPVC.LastSyncDuration.After(maxLastSyncDuration.Time) {
```

Comment:
1. The first part test is unnecessary.  It is already done in line 1160.
2. In the second part, you are using `After`. Shouldn't you use `Before`? So if assume the following
`protectedPVC.LastSyncDuration = 2022`
`maxLastSyncDuration = 2023`
We want `maxLastSyncDuration` to be updated and refer to time 2022, right? So we should be using `Before`

Reply:
so with the latest update, we longer will be able to use `After` or `Before` because the type of the `LastSyncDuration` is not `time` but `Duration` and this does not have `After`, the underlying type of `Duration` is `int64` and we can use `>` to compare